tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -1594,14 +1594,15 @@ public function provide70TypesOrderAndLength()
      * @dataProvider provide70TypesOrderAndAlphabet
      * @requires PHP 7.0
      *
-     * @param mixed $expected
-     * @param mixed $input
+     * @param string      $expected
+     * @param string|null $input
+     * @param string[]    $importOrder
      */
-    public function test70TypesOrderAndAlphabet($expected, $input = null)
+    public function test70TypesOrderAndAlphabet($expected, $input = null, array $importOrder = null)
```

Comment:
prep. for missing tests for case described here https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2696